src/main/resources/assets/constructionwand/lang/ja_jp.json

```diff
@@ -0,0 +1,70 @@
+  "constructionwand.option.lock.northsouth": "§6南北方向",
+  "constructionwand.option.lock.northsouth.desc": "目標ブロックの上に南北方向に設置します。",
+  "constructionwand.option.lock.eastwest": "§6東西方向",
+  "constructionwand.option.lock.eastwest.desc": "目標ブロックの上に東西方向に設置します。",
+  "constructionwand.option.lock.nolock": "§cなし",
+  "constructionwand.option.lock.nolock.desc": "目標ブロックからどんな方向にも拡張できます。",
+
+  "constructionwand.option.direction": "設置する向き: ",
+  "constructionwand.option.direction.target": "§6目標ブロック基準",
+  "constructionwand.option.direction.target.desc": "目標ブロックと同じ向きに設置します。",
+  "constructionwand.option.direction.player": "§aプレイヤー基準",
+  "constructionwand.option.direction.player.desc": "プレイヤーに向かって設置します。",
+
+  "constructionwand.option.replace": "ブロックの置換: ",
+  "constructionwand.option.replace.yes": "§a置換する",
+  "constructionwand.option.replace.yes.desc": "液体、雪、背の高い草など特定のブロックを置換します。",
+  "constructionwand.option.replace.no": "§c置換しない",
+  "constructionwand.option.replace.no.desc": "ブロックを置換しません。",
+
+  "constructionwand.option.match": "適合条件: ",
+  "constructionwand.option.match.exact": "§a完全一致",
+  "constructionwand.option.match.exact.desc": "厳密に一致するブロックのみが拡張できます。",
+  "constructionwand.option.match.similar": "§6類似",
+  "constructionwand.option.match.similar.desc": "土ブロックと草ブロックなど、似たブロックを同じものとして扱います。",
+  "constructionwand.option.match.any": "§c全て",
+  "constructionwand.option.match.any.desc": "どんなブロックも拡張できます。",
+
+  "constructionwand.option.random": "ランダム設置: ",
+  "constructionwand.option.random.yes": "§aランダム設置する",
+  "constructionwand.option.random.yes.desc": "ホットバー内にあるブロックをランダムに設置します。",
+  "constructionwand.option.random.no": "§cランダム設置しない",
+  "constructionwand.option.random.no.desc": "ランダム設置を行いません。",
+
+  "constructionwand.description.wand": "%1$sは、向かっている面に%2$dブロック設置でき、%3$s使用できます。\n\n%5$を押しながらホイールスクロールすると設置方向(水平、垂直、南北、東西、無制限)を変更できます。\n\n%6$s§9を押しながら使用§0でオプション画面を開きます。\n\n§5§n取消§0§r\nブロックの方を向きながら§9スニークしながら§0%4$sを押し続けると、最後に設置したブロックが緑の境界線ととともに表示されます。§9スニークしながら§0%4$s§9を押して使用ボタンを押す§0と、処理を取り消すことができ、すべてのブロックはインベントリに戻ります。破壊の杖を使用していた場合もブロックはもとに戻ります。\n\n§5§コンテナ§0§r\nシュルカーボックス、バンドル、その他のModで追加されたコンテナは杖で建築を行った際に資材を供給できます。\n\n§5§nオフハンド優先§0§r\nオフハンドにブロックを持っている場合、設置されているブロックよりそちらが優先されます。",
+  "constructionwand.description.durability.limited": "残り%dブロック",
+  "constructionwand.description.durability.unlimited": "無制限に",
+  "constructionwand.description.key.sneak": "スニーク",
+  "constructionwand.description.key.sneak_opt": "スニーク+%s",
+  "constructionwand.description.core": "§5§nセッティング§0§r\n新しいコアと杖を一緒にクラフトが麺に設置することでセッティングできます。コアを交換するためには、杖を持って何もない空間を%sを押しながら攻撃するか、オプション画面を使用してください。",
+  "constructionwand.description.core_angel": "天使のコアはブロックの反対側にブロックを設置します。最大距離は杖の段階に依存します。オフハンドにブロックを持った状態で、何もない空間に使用すると空中にそのブロックを設置できます。",
+  "constructionwand.description.core_destruction": "破壊のコアは正面のブロック(タイルエンティティは不可)を破壊できます。破壊できる最大ブロック数は杖の段階に依存します。破壊されたブロックは消滅しますが、間違えたときは取り消すことができます。",
+
+  "stat.constructionwand.use_wand": "杖を用いてブロックを設置しました。"
```

Comment:
This seems to be a full sentence (with a full stop). Please reword, since it is just the label for a MC statistic

Reply:
Oh, I see. Certainly, this is full sentence in Japanese.
I fixed it.